package.json

```diff
@@ -14,7 +14,7 @@
     "url": "https://jira.sonarsource.com/browse/SLVSCODE",
     "email": "sonarlint@googlegroups.com"
   },
-  "license": "LGPL-3.0",
+  "license": "SEE LICENSE IN LICENSE.txt",
```

Comment:
Note that this page is specific to vscode, not sure it is standard for all npm modules.